sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/compute/samples/CloneVirtualMachineToNewRegion.java

```diff
@@ -0,0 +1,284 @@
+                .withPrimaryPrivateIPAddressDynamic()
+                .withNewPrimaryPublicIPAddress(publicIpDnsLabel)
+                .withPopularLinuxImage(KnownLinuxVirtualMachineImage.UBUNTU_SERVER_18_04_LTS)
+                .withRootUsername(userName)
+                .withSsh(sshPublicKey)
+                .withNewDataDisk(100)
+                .withNewDataDisk(100, 1, CachingTypes.READ_WRITE)
+                .withSize(VirtualMachineSizeTypes.fromString("Standard_D2a_v4"))
+                .create();
+
+            System.out.println("Created a Linux VM with managed OS and data disks: " + linuxVM.id());
+            Utils.print(linuxVM);
+
+            // Gets the specialized managed OS and Data disks of the virtual machine
+            //
+            Disk osDisk = azureResourceManager.disks().getById(linuxVM.osDiskId());
+            List<Disk> dataDisks = new ArrayList<>();
+            for (VirtualMachineDataDisk disk : linuxVM.dataDisks().values()) {
+                Disk dataDisk = azureResourceManager.disks().getById(disk.id());
+                dataDisks.add(dataDisk);
+            }
+
+            //=============================================================
+            // Deallocating the virtual machine
+            System.out.println("Deallocating VM: " + linuxVM.id());
+
+            linuxVM.deallocate();
+
+            System.out.println("Deallocated the VM");
+
+            //=============================================================
+            // Create incremental Snapshot from the OS managed disk
+
+            System.out.printf("Creating managed snapshot from the managed disk (holding specialized OS): %s %n", osDisk.id());
+
+            Snapshot osSnapshot = azureResourceManager.snapshots()
+                .define(osDisk.name() + "-snp")
+                .withRegion(region)
+                .withExistingResourceGroup(rgName)
+                .withLinuxFromDisk(osDisk)
+                .withIncremental(true) // incremental is mandatory for snapshot to be copied across region
+                .create();
+
+            //=============================================================
+            // Copy snapshots to the new region
+
+
+            System.out.printf("Copying managed snapshot %s to a new region.%n", osDisk.id());
+
+            Snapshot osSnapshotNewRegion = azureResourceManager
+                .snapshots()
+                .define(osDisk.name() + snapshotCopiedSuffix)
+                .withRegion(regionNew)
+                .withNewResourceGroup(rgNameNew)
+                .withDataFromSnapshot(osSnapshot)
+                .withCopyStart()
+                .withIncremental(true)
+                .create();
+            osSnapshotNewRegion.awaitCopyStartCompletion();
+
+            // Delete snapshot from old region
+            azureResourceManager.snapshots().deleteById(osSnapshot.id());
+
+            System.out.println("Created managed snapshot holding OS: " + osSnapshotNewRegion.id());
```

Comment:
nit: use `System.out.printf` to make it consistent with other output statement with parameter?

Reply:
Changed to `printf`. Left `println` without parameters.